server/src/main/scala/drt/server/feeds/lgw/LGWForecastXLSExtractor.scala

```diff
@@ -58,7 +58,7 @@ object LGWForecastXLSExtractor {
         val serviceCell = stringCellOption(headingIndexByNameMap("Service"), row)
         val arrivalOrDepCell = stringCellOption(headingIndexByNameMap("ArrDep"), row)
         val internationalDomesticCell = stringCellOption(headingIndexByNameMap("Dom/Int"), row)
-        val totalCell = numericCellOption(headingIndexByNameMap("Forecast Pax"), row)
+        val totalCell = tryNumericThenStringCellDoubleOption(headingIndexByNameMap("Forecast Pax"), row)
```

Comment:
seeing as you are converting this to an Int anyway, maybe it would be better to use the Int function rather than the double one?

Reply:
👍